LICENSE.txt

```diff
@@ -19,10 +19,3 @@ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
 ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-
-Other authors retain ownership of their contributions. If a submission can
-reasonably be considered independently copyrightable, it's yours and we
-encourage you to claim it with appropriate copyright notices. This submission
-then falls under the "otherwise noted" category. All submissions are strongly
-encouraged to use the two-clause BSD license reproduced above.
```

Comment:
Since this has been moved to `CONTRIBUTING.txt`, should the copyright lines above also be removed? After all we say that the ownership of contributions is indicated by the git history.

Reply:
Do you mean the M-Labs copyright notice? I think that needs to stay, according to point 1. of license. 

You could _add_ to the that list of copyright holders. I was a little surprised your name isn't there already, but I don't understand all the history.

Digression, doesn't have to be resolved in this change: 

(IANAL, but) my understanding is that the exact names in the list on the license will have little effect on the ways in which nMigen may be used - each and every contributor owns the copyright for their contributions, and each and every contributor has licensed their contribution under this license text. 

Where the list of copyright holders in the binary does have more of an effect is with giving credit when the source or binaries are distributed without git history. For instance, if someone were to use nMigen in their binary, the list of copyright holders might be the only remaining record of who wrote nMigen. Similar kind of situation if someone redistributed nMigen in a zip file.

I don't know how you feel about these scenarios, but I'd be little outraged on your behalf if there was a distribution of nMigen without whitequark's name.

Reply:
Ah right. I think `Copyright (C) 2019-2020 whitequark` should be added then.

Reply:
Done!